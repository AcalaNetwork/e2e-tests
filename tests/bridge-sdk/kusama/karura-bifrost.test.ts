import { bridgeSDKTest } from '../shared'

bridgeSDKTest({
  from: 'karura',
  to: 'bifrost',
  token: 'KUSD',
})
