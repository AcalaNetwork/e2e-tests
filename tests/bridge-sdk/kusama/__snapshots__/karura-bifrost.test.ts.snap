// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`'karura' to 'bifrost' using bridgeSDK cross-chain 'KUSD' > after 1`] = `
{
  "address": "5FA9nQDVg267DEd8m1ZypXLBnvN7SFxYwV7ndqSYGiN9TTpu",
  "decimals": 12,
  "fromChain": 8,
  "toChain": "(rounded 1.97)",
}
`;

exports[`'karura' to 'bifrost' using bridgeSDK cross-chain 'KUSD' > fee 1`] = `"(rounded 0.03)"`;

exports[`'karura' to 'bifrost' using bridgeSDK cross-chain 'KUSD' > initial 1`] = `
{
  "address": "5FA9nQDVg267DEd8m1ZypXLBnvN7SFxYwV7ndqSYGiN9TTpu",
  "decimals": 12,
  "fromChain": 10,
  "toChain": 0,
}
`;
